sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/ParserUtils.scala

```diff
@@ -71,6 +71,12 @@ object ParserUtils {
     stream.getText(interval)
   }
 
+  /** Get all the text which between the given start and end tokens. */
+  def interval(start: Token, end: Token): String = {
```

Comment:
this seems hacky. I think the named solution(`anything`) is better.

Reply:
https://github.com/apache/spark/pull/30332/files#diff-4ed0783e637633458d3b2e929553c134b5e2395785ec540b5347c64916c3d686R179-R180
For the case above, `anything` converts "a b" -> "ab"

Reply:
interesting... Can you add a comment to explain it?

Reply:
OK~